tools/setup_gpu_build_tools.sh

```diff
@@ -23,11 +23,22 @@
 # the following environment variables:
 # PATH, CPLUS_INCLUDE_PATH, C_INCLUDE_PATH, LIBRARY_PATH, LD_LIBRARY_PATH, NVCC
 
+set -e
+
 VARIANT=$1
 DEPS_PATH=$2
 
 >&2 echo "Setting CUDA versions for $VARIANT"
-if [[ $VARIANT == cu102* ]]; then
+if [[ $VARIANT == cu110* ]]; then
+    CUDA_VERSION='11.0.221-1'
+    CUDA_PATCH_VERSION='11.2.0.252-1'
+    CUDA_LIBS_VERSION='10.2.1.245-1'
+    CUDA_SOLVER_VERSION='10.6.0.245-1'
+    CUDA_NVTX_VERSION='11.0.167-1'
+    LIBCUDA_VERSION='450.51.06-0ubuntu1'
```

Comment:
is this the minimum required version? @DickJC123

Reply:
I think you can use 450.36.06 here instead. @waytrue17

Reply:
@szha Updated the version, please review again, thanks!